apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -80,6 +145,74 @@ const editOrSendMessage = async (
   }
 };
 
+const generateProofUrl = (
```

Comment:
Moved from the TelegramService class and now takes validEventIds as an arg, which are populated when a user chooses which group they want to join.